animal.go

```diff
@@ -110,49 +155,16 @@ func (g game) LearnNewAnimal(r io.Reader, w io.Writer, pq string) {
 
 }
 
-// StartingData ...
-var StartingData = map[string]Question{
-	"Does it have 4 legs?": Question{
-		Yes: "Does it have stripes?",
-		No:  "Is it carnivorous?",
-	},
-	"Does it have stripes?": Question{
-		Yes: "Is it a zebra?",
-		No:  "Is it a lion?",
-	},
-	"Is it carnivorous?": Question{
-		Yes: "Is it a snake?",
-		No:  "Is it a worm?",
-	},
-	"Is it a zebra?": Question{
-		Yes: AnswerWin,
-		No:  AnswerLose,
-	},
-	"Is it a giraffe?": Question{
-		Yes: AnswerWin,
-		No:  AnswerLose,
-	},
-	"Is it a lion?": Question{
-		Yes: AnswerWin,
-		No:  AnswerLose,
-	},
-	"Is it a snake?": Question{
-		Yes: AnswerWin,
-		No:  AnswerLose,
-	},
-	"Is it a worm?": Question{
-		Yes: AnswerWin,
-		No:  AnswerLose,
-	},
-}
-
-// StartingQuestion ...
-var StartingQuestion = "Does it have 4 legs?"
-
-// Question ...
-type Question struct {
-	Yes string // what question is next, if answer is yes
-	No  string // what question is next, if answer is no
+// Replay ... to replay the game
+func Replay() bool {
+	fmt.Print("Would you like to play again (y/n)? ")
+	var replay string
+	_, _ = fmt.Fscanln(os.Stdin, &replay)
```

Comment:
We don't need to use `Fscanln` if we're reading from `os.Stdin`, since that's what `Scanln` reads from already. (If we wanted to unit test this function, we would need to pass in the reader, but I don't think we need bother.)